web/src/components/dateTime.js

```diff
@@ -1,4 +1,3 @@
-import {Typography} from '@material-ui/core';
 import React, {Component} from 'react';
```

Comment:
File name should only include lower case.

"File names must be all lowercase and may include underscores (_) or dashes (-), but no additional punctuation. Follow the convention that your project uses. Filenames’ extension must be .js."